src/elements/common/messages.js

```diff
@@ -858,6 +858,11 @@ const messages = defineMessages({
         defaultMessage: 'Decline',
         description: 'Reject option for a task',
     },
+    taskDueDateLabel: {
+        id: 'be.tasks.taskDueDate',
+        defaultMessage: 'Due: ',
```

Comment:
Is this the right way to translate something like "Due: Jan 19 at 2pm"? It seems like the position of the date string may vary across languages and RTL setting.

Reply:
Agreed, this seems a bit odd. Can we use a variable for the date, at least?

Reply:
this is actually true for the previously used translation, since it has no label separator. in this case, "Due: " functions as a field label so I feel this is fine. 

The other field is part of a sentence structure, and should actually be one full message instead of two. I didn't change it to avoid complicating this review

Reply:
Fair point on the ordering; this will always put the label to the left of the date because of the two components. I can fix this